### PR TITLE
Feature/hera 6 create profile card

### PR DESCRIPTION
**Pull request by julia-gatina**
Created profile card, but forgot to add HERA-6 to commit messages. 

tweeter-profile-card

### CODE REVIEW

File: src/app/shared/components/dashboard/profile/profile.component.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+:host {
+
+  .profile {
+    margin-top: 110px;
+    height: 400px;
+    background-color: rgb(155, 157, 157);
+    background-image: url("/assets/brick-wall.png");
+    color: #545149;
```
Comment (RenatGatin) [inline src/app/shared/components/dashboard/profile/profile.component.scss:8]: Don't use #hash colors in component's SCSS.
But instead:
1) create _colors.scss in the /src/styles
image
2) add color name for this #hash using web-site https://www.color-blindness.com/color-name-hue/ 
example for this color name is: $masala-brown
3) import _colors.scss into your SCSS and use this color by name
image

**Review by RenatGatin — CHANGES_REQUESTED** [review]
fix color usage in SCSS

File: src/app/shared/components/dashboard/profile/profile.component.scss (hunk 1 above)
Comment (julia-gatina) [reply src/app/shared/components/dashboard/profile/profile.component.scss]: Done 
Screen Shot 2022-08-25 at 12 33 05 PM
Screen Shot 2022-08-25 at 12 32 48 PM

File: src/app/shared/components/navigation/navigation.component.scss [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
+:host {
   .navbar {
-    background-color: #99c435;
+    background-color: $yellow-green-green;
     height: 120px;
-    box-shadow: 2px 2px 10px #397207;
-    text-shadow: 2px 2px 3px #307207;
+    box-shadow: 2px 2px 10px green;
+    text-shadow: 2px 2px 3px green;
```
Comment (RenatGatin) [inline src/app/shared/components/navigation/navigation.component.scss:8]: All colors of the Application should be declared in the single place ("src/styles/colors"). 
So don't use native "green" color here, but declare corresponding to native "green" #hash color in "src/styles/colors" and reference it here.

**Review by RenatGatin — CHANGES_REQUESTED** [review]
Fix "green" color (see comments)

File: src/app/shared/components/navigation/navigation.component.scss (hunk 2 above)
Comment (julia-gatina) [reply src/app/shared/components/navigation/navigation.component.scss]: Fixed 'green' and other colors declarations and references. .

**Review by RenatGatin — APPROVED** [review]
Умничка!

**Pull request merged**